py-polars/polars/series/series.py

```diff
@@ -3528,6 +3532,16 @@ def search_sorted(
         ----------
         element
             Expression or scalar value.
+            If this value matches the dtype of values in self, the return result is an
+            integer.
+            If self's dtype is ``pl.List``/``pl.Array``, we assume an element that is
+            ``list`` or NumPy array is a single value, and return an integer. (For other
+            dtypes, a ``list``/NumPy array element is assumed to be searching for
+            multiple values, and the return result is a ``Series``; this is deprecated,
+            you should switch to using a ``Series`` if you want to search for multiple
+            values.)
+            If this is a ``Series`` or ``Expr``, the return result is a ``Series``.
```

Comment:
This can be removed now I think.

Reply:
That's still an accurate description of behavior though? Or is it that you don't want to document the deprecated behavior since people should not be using it?

Reply:
This is being deprecated by this PR, right? Maybe it should be in a `..warning` or `..note` then.